Kentico/kontent-boilerplate-net
# DEL-2348 Transform ActionFilter into middleware

### Motivation

Fixes #84 

### Checklist


### How to test

Set up webhooks using ngrok, fire notification in Kentico Kontent and check that the SignatureMiddleware validates notification hash correctly.
